### PR TITLE
removed image url for none url images

### PR DESCRIPTION
**Pull request by chitalian**

### CODE REVIEW

File: worker/src/lib/dbLogger/logResponse.ts [hunk 1]
```diff
@@ -77,6 +77,37 @@ async function getPromptId(
...
+  if (typeof body !== "object" || body === null) {
+    return body;
+  }
+  if (Array.isArray(body)) {
+    return body.map((item) => unsupportedImage(item));
+  }
+  if (body["image_url"] !== undefined) {
+    const imageUrl = body["image_url"];
+    if (typeof imageUrl === "string" && !imageUrl.startsWith("http")) {
+      body.image_url = { unsupported_image: true };
```
Comment (colegottdank) [inline worker/src/lib/dbLogger/logResponse.ts:93]: Should this be returned? Or we will be recursively calling this

Comment (colegottdank) [reply worker/src/lib/dbLogger/logResponse.ts:93]: Same with where you set body.image_url below

Comment (colegottdank) [reply worker/src/lib/dbLogger/logResponse.ts:93]: return { ...body, image_url: { unsupported_image: true } };

File: worker/src/lib/HeliconeProxyRequest/handler.ts [hunk 2]
```diff
@@ -71,6 +71,15 @@ export async function handleProxyRequest(
   responseHeaders.set("Helicone-Status", "success");
   responseHeaders.set("Helicone-Id", proxyRequest.requestId);
 
+  let status = response.status;
+  if (status < 200 || status >= 600) {
+    console.error("Invalid status code: ", status);
+    status = 200;
```
Comment (chitalian) [inline worker/src/lib/HeliconeProxyRequest/handler.ts:77]:
```suggestion
    status = 500;
```

**Pull request merged**